content/chainguard/chainguard-images/images-testing.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "How Chainguard Images are Tested"
+linktitle: "How Images are Tested"
+type: "article"
+description: "A conceptual article outlining testing requirements for Chainguard Images."
+date: 2024-03-13T11:07:52+02:00
+lastmod: 2023-03-13T11:07:52+02:00
+draft: false
+tags: ["Overview", "Chainguard Images", "Product"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 550
+toc: true
+---
+
+Chainguard Images are minimal, distroless container images that you can use to build and run secure applications. A frequently-asked question about these Images relates to what kind of testing Chainguard performs on them to ensure they match the functionality of their upstream counterparts.
+
+This article provides a high level overview of what testing Chainguard performs when building new Images to ensure their security and that they provide an experience consistent with their external counterparts.
+
+
+## Build requirements for new Images
+
+Chainguard has a set of requirements in place that new Images must meet in order to be included in our library of Images. These requirements fall into two categories:
+
+* Image standards
+* Application testing
+
+
+### Image standards
+
+When building a new Image, Chainguard will take steps to ensure it meets the following standards:
+
+| **Requirement** 	  |  **Explanation**     |
+| --- | --- |
+| **Size**     |  Any new Chainguard Images should be smaller than their external counterparts. Any Images that aren't smaller than their counterparts must include an explanation as to why.     |
```

Comment:
I'm not totally sure that we do this today in terms of the explanation, it's not in readmes that are larger sizes, for example

Reply:
I'll see if I can get some clarification on this but for now I'll just remove that sentence.